Two95JavaTraining/Sujit/assignment8_Threads/NetworkPrinter.java

```diff
@@ -0,0 +1,60 @@
+package assignment8_Threads;

```

Comment:
package name should follow the sun java conventions for eg: com.java.learning.threads.assignment makes more sense.